<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TX717-4441WGD" name="Default test" tests="1" failures="1" timestamp="12 Apr 2017 14:42:28 GMT" time="0.031" errors="0">
  <testcase name="runTC_SF_ManagerLogin" time="0.015" classname="testscripts.SF.SmokeTest_Manager.TC_SF_ManagerLogin">
    <failure type="org.testng.TestNGException" message="
The data provider is trying to pass 5 parameters but the method testscripts.SF.SmokeTest_Manager.TC_SF_ManagerLogin#runTC_SF_ManagerLogin takes 1">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 5 parameters but the method testscripts.SF.SmokeTest_Manager.TC_SF_ManagerLogin#runTC_SF_ManagerLogin takes 1
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1229)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1100)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- runTC_SF_ManagerLogin -->
</testsuite> <!-- Default test -->
